Samples/tutorials/excel-tutorial/My Office Add-in/src/taskpane/taskpane.js

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+ * See LICENSE in the project root for license information.
+ */
+
+/* global console, document, Excel, Office */
+
+Office.onReady((info) => {
+  if (info.host === Office.HostType.Excel) {
+    // Determine if the user's version of Office supports all the Office.js APIs that are used in the tutorial.
+    if (!Office.context.requirements.isSetSupported('ExcelApi', '1.7')) {
+      console.log('Sorry. The tutorial add-in uses Excel.js APIs that are not available in your version of Office.');
```

Comment:
tutorial probably shouldn't enable the buttons if the APIs are unavailable.

Reply:
TBH, I think the tutorial shouldn't include this `isSetSupported` stuff. If you're targeting something this old, you have a very specific audience in mind. Thoughts?